nirc_ehr/resources/queries/study/necropsy.js

```diff
@@ -31,8 +31,10 @@ function onUpsert(helper, scriptErrors, row, oldRow) {
 
     if (!helper.isETL()) {
 
-        if (deathIdMap[row.Id] && deathIdMap[row.Id].QCStateLabel.toUpperCase() === 'REQUEST: PENDING' ||
-                deathIdMap[row.Id].QCStateLabel.toUpperCase() === 'REVIEW REQUIRED') {
+        if (deathIdMap[row.Id] && deathIdMap[row.Id].QCStateLabel &&
+                (deathIdMap[row.Id].QCStateLabel.toUpperCase() === 'IN PROGRESS' ||
```

Comment:
I don't think we need to do strict error checking for saving a draft.

Reply:
good point, removed